report/custom.go

```diff
@@ -276,37 +274,32 @@ func (d *DiffCustomMetricSet) Table() string {
 	table.SetCenterSeparator("|")
 	table.SetColumnAlignment([]int{tablewriter.ALIGN_LEFT, tablewriter.ALIGN_RIGHT, tablewriter.ALIGN_RIGHT, tablewriter.ALIGN_RIGHT})
 	table.SetHeader([]string{"", makeHeadTitleWithLink(d.B.report.Ref, d.B.report.Commit, nil), makeHeadTitleWithLink(d.A.report.Ref, d.A.report.Commit, nil), "+/-"})
+	report := d.report
+	if report == nil {
+		report = &Report{}
+	}
+
 	for _, m := range d.Metrics {
 		var va, vb, diff string
 		switch {
 		case m.A == nil && m.B == nil:
 			continue
 		case m.A != nil && m.B == nil:
 			vb = ""
-			if isInt(*m.A) {
-				va = fmt.Sprintf("%d%s", int(*m.A), m.customMetricA.Unit)
-				diff = fmt.Sprintf("%d%s", int(m.Diff), m.customMetricA.Unit)
-			} else {
-				va = fmt.Sprintf("%.1f%s", *m.A, m.customMetricA.Unit)
-				diff = fmt.Sprintf("%.1f%s", m.Diff, m.customMetricA.Unit)
-			}
+			va = fmt.Sprintf("%s%s", report.convertFormat(*m.A), m.customMetricA.Unit)
+			diff = fmt.Sprintf("%s%s", report.convertFormat(m.Diff), m.customMetricA.Unit)
 		case m.A == nil && m.B != nil:
 			va = ""
-			if isInt(*m.B) {
-				vb = fmt.Sprintf("%d%s", int(*m.B), m.customMetricB.Unit)
-				diff = fmt.Sprintf("%d%s", int(m.Diff), m.customMetricB.Unit)
-			} else {
-				vb = fmt.Sprintf("%.1f%s", *m.B, m.customMetricB.Unit)
-				diff = fmt.Sprintf("%.1f%s", m.Diff, m.customMetricB.Unit)
-			}
+			va = fmt.Sprintf("%s%s", report.convertFormat(*m.B), m.customMetricB.Unit)
```

Comment:
```suggestion
			vb = fmt.Sprintf("%s%s", report.convertFormat(*m.B), m.customMetricB.Unit)
```

It seems that an error was made in the static analysis because it was mistakenly assigned to va.